### PR TITLE
chore(package): bump fxa-shared to 1.0.14 + npm shrinkwrap

### PR DESCRIPTION
**Pull request by philbooth**
@mozilla/fxa-devs r?

### CODE REVIEW

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -262,7 +269,7 @@
       "dependencies": {
         "acorn": {
           "version": "3.3.0",
-          "resolved": "https://registry.npmjs.org/acorn/-/acorn-3.3.0.tgz",
+          "resolved": "http://registry.npmjs.org/acorn/-/acorn-3.3.0.tgz",
```
Comment (shane-tomlinson) [inline npm-shrinkwrap.json]: Lots of `http` resolutions, are you using the latest npm?

Comment (philbooth) [reply npm-shrinkwrap.json]:
```
$ npm --version
6.4.1
```

Comment (vladikoff) [reply npm-shrinkwrap.json]: https://blog.npmjs.org/post/154400916805/avoid-http-urls-in-shrinkwrap-files

for some reason they couldn't fix that on their end 🤷‍♂️

Comment (philbooth) [reply npm-shrinkwrap.json]: Ref: #2641

**Pull request merged**